timetracker/entries/forms.py

```diff
@@ -1,23 +1,35 @@
 from django import forms
 from django.utils import timezone
 
-from .models import Project, Client
+from .models import Project, Client, Entry
 
 
-class ClientForm(forms.Form):
-    name = forms.CharField()
+class ClientForm(forms.ModelForm):
+    class Meta:
+        model = Client
+        fields = ('name',)
 
+class ProjectForm(forms.ModelForm):
+    class Meta:
+        model = Project
+        fields = ('name', 'client')
 
-class ProjectForm(forms.Form):
-	name = forms.CharField()
-	client = forms.ModelChoiceField(queryset=Client.objects.all())
+class EntryForm(forms.ModelForm):
+    class Meta:
+        model = Entry
+        fields = ('start', 'stop', 'project', 'description')
+        help_texts = {
```

Comment:
You might want to look into moving the help text to the respective field on the model. That will ensure the help text is there each time you have a model form of this model.